react/components/ShippingInfo.tsx

```diff
@@ -8,16 +8,16 @@ interface CustomProps {
 
 const ShippingInfo: FunctionComponent<CustomProps> = ({ option }) => {
   return (
-    <div className="w-100">
-      <p className="flex flex-row justify-between mv3">
-        <span>{option.id}</span>{' '}
-        <span>
-          <FormattedCurrency value={option.value / 100} />
-        </span>
-      </p>
-      <p className="c-muted-1 mv3">
-        <TranslateEstimate shippingEstimate={option.estimate} />
-      </p>
+    <div className="flex w-100">
+      <div className="flex-auto">
+        <div className="mb3">{option.id}</div>
```

Comment:
Why have you changed from `p` to `div`, I mean, why was that necessary?

Reply:
I think @omninando can answer you.

Reply:
Actually, I just remembered it was @augustocb . He told me something about `p` tag semantics.

Reply:
Two points:
- `p` is a semantic tag for *paragraphs*
- `p` has a margin by default, that is not necessarily the expected one